cmd/release/command.go

```diff
@@ -0,0 +1,56 @@
+package release
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+)
+
```

Comment:
Problem 2: How do you do changes to CHAGELOG.md?

Reply:
https://github.com/giantswarm/devctl/pull/48/#issuecomment-610309949

Reply:
I have mixed feelings about "up to each" team approach. I worry we split too much too soon if we foster that thinking. At least when it comes to the codebases. I'd like to that part to be rather aligned. This is also important when moving from team to team. CC @tuommaki.

Reply:
How can we make it standard for every team?

Reply:
Agreed, I missed the news about not maintaining CHANGELOG.md files, and I rather like the practise.

Reply:
Extending the changelog with the link in the markdown should be doable in code as well. Would even make it way more reliable because the changelog links are already a little different here and there because humans.

Reply:
> Agreed, I missed the news about not maintaining CHANGELOG.md files, and I rather like the practise.

After this discussion, I'm super confused. Should they be maintained or not? What tooling uses them and what ensures they are maintained? Yesterday I tried to find some source of truth for the advice of not maintaining versionbundle changelogs, but could only refer the PR comments.

When we need CHANGELOG maintained, having it built-in tooling is important so that it's consistent.

I personally feel that we should automate as much as possible with the release process to avoid problems. And currently we do have problems, because so many things change frequently and it's very difficult to follow and remember every snippet of news post. As I wrote the other day, it's even difficult to find relevant documentation, though this gets improved in a bit.

Reply:
@tuommaki I think the source of truth for changelogs should be CHANGELOG.md inside the repo. I'm actually quite big on that. To me it doesn't make sense to maintain the changelog outside the repository because it requires extra PR to a separate repo to update it.

One day I talked with @MarcelMue and he said we potentially can link to changelogs inside operator repositories which would be my preference from the development perspective.

I agree there should be more automation around that. And there is work being done towards that direction. There is even `architect/changelog` job. For now it only validates the changelog.

In my ideal world a GitHub release should be created by the CI after new changelog version header is added and the version is somehow injected into the build.